GameServerLib/GameObjects/AttackableUnits/AI/LaneMinion.cs

```diff
@@ -81,6 +81,7 @@ public override void Update(float diff)
             {
                 if (IsDashing || _aiPaused)
                 {
+                    Replication.Update();
```

Comment:
Isn't this called anyway at https://github.com/LeagueSandbox/GameServer/pull/813/files#diff-34249f2c499de7e41150dcb60a310775R100 ?

Reply:
Alternatively, the replicate could be called outside of the update method completely, somewhere which is guaranteed to be run regardless of what happens in update. A new function called `PostUpdate()` maybe?

Reply:
I'm not entirely sure the if statement and return is even necessary, why would we not want it to update if paused or dashing in the first place?